spec/spec_helper.rb

```diff
@@ -1,13 +1,96 @@
-require 'support/simplecov'
-
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```

Comment:
Why did you add these comments? Remove them.

Why did you remove this line require 'support/simplecov' ?